tools/packaging/common/sidecar.env

```diff
@@ -102,3 +101,5 @@ OUTPUT_CERTS=/etc/certs
 # and returning workload certificates. Istiod is implementing the protocol, and is the default value
 # if ISTIO_CA is not set.
 #ISTIO_CA
+# set CA_ADDR if your istiod.istio-system.svc is on port other than 15102
+#CA_ADDR=istiod.istio-system.svc:32018
```

Comment:
NIT: We need to document CA port here https://istio.io/docs/ops/deployment/requirements/#ports-used-by-istio.

Reply:
Also, where in the code do we use CA_ADDR?

Reply:
It is used in pkg/istio-agent/sds-agent.go:L91:
	`caEndpoint = "CA_ADDR"`

Reply:
What is the default CA_ADDR? I saw the in pkg/istio-agent/sds-agent.go has "serverOptions.CAEndpoint = "istiod.istio-system.svc:15012".

Reply:
yes, this is the default value

Reply:
oh I saw this is commented out. So it's just an example.

Reply:
yes